dts/bindings/xen,domu-vbd.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2024 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xen DomU configuration for virtual block device
+compatible: "xen,domu-vbd"
+
+include: base.yaml
+
+properties:
+  backend_domain_id:
+    type: int
+    default: 1
```

Comment:
Note Zephyr policy "In any case where default: is used in a devicetree binding, the description: for that property must explain why the value was selected and any conditions that would make it necessary to provide a different value."

As for me, it might be better to: (a) define default in code (b) make them "required"

Reply:
It was done solely to reduce the number of lines in the DTS. With the current implementation, most of the devices' backends are static in dom@1. I thought it would be simple for those familiar with the topic to understand, so I didn't think it was worth mentioning.

So, in rare cases when we have different backend IDs, we simply add this configuration to the DTS.

> (b) make them "required"

In this case, making them 'required' renders 'default' meaningless because we always need to add a property to the node.

In any case, I believe this comment is no longer relevant due to comments from @lorc about new implementation

Reply:
Yeah. This property will go away completely.

Reply:
It is not about this property  - it is about ALL properties for which you've added "default: X". Those must have description with reasoning as per above **Zephyr code policy** I've quoted above.

So, please (a) update bindings for all   "default: X" (b) drop defaults from bindings and move them in code.